Java/Symbol List%3A Constants.tmPreferences

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<plist version="1.0">
+<dict>
+	<key>name</key>
+	<string>Symbol List: Constants</string>
+	<key>scope</key>
+	<string>(source.java) &amp; (entity.name.constant)</string>
+	<key>settings</key>
+	<dict>
+		<key>showInIndexedSymbolList</key>
```

Comment:
Do you only want these to be indexed, not in the symbol list? If they should be in the symbol list, we'll need to add `showInSymbolList` with a value in `1`.